### PR TITLE
Remove `/proxy` endpoint, #26930

### PR DESCRIPTION
**Pull request by Joozty**
https://is.roihunter.com/issues/26930

Waiting for merging https://github.com/business-factory/lady-image/pull/630 and changing endpoint on frontend side. 

@business-factory/pythonistas Please review but do not merge :)

### CODE REVIEW

File: hooks/api_server.py [hunk 1]
```diff
@@ -71,54 +71,12 @@ def on_post(self, req, resp):
...
-        resp.set_header("access-control-max-age", "21600")
-        allow_headers = req.get_header("access-control-request-headers")
-        if allow_headers:
-            resp.set_header("access-control-allow-headers", allow_headers)
-
-        url = req.get_param("url")
-
-        try:
-            self._logger.info("Trying to proxy URL: %s", url)
-            response = requests.get(url, timeout=self._TIMEOUT_IN_SECONDS, headers={"User-Agent": self._get_user_agent(req)})
```
Comment (harpener) [inline hooks/api_server.py]: @Joozty I think you removed the dependency on `requests`. :-) Remember also the eventlet patch.

Comment (Joozty) [reply hooks/api_server.py]: https://github.com/business-factory/captain-hook/pull/39/commits/e431e335dd199202560b92b6354dcb282fa1f9c2

Comment (Joozty) [reply hooks/api_server.py]: https://github.com/business-factory/captain-hook/pull/39/commits/35ea0931bd0ff9dc694d01ac37677bb809bde56b

**Comment by harpener** [comment]
I've honestly forgot this is still open. I don't know the flow in TEDI, but it is still not on production there. @miso-belica @Joozty Also, I found a small reminder in comment there :-) https://github.com/business-factory/tedi/pull/246/files#diff-3dce86cb0e10781bf1b25a7505dc20b7R327

**Comment by Joozty** [comment]
@harpener Don't worry I check it from time to time and if changes on TEDI will be on master I will ping you. :) There are still requests coming to Captain Hook. 

https://logs.roihunter.com/streams/5788dfc71a89614ec95ee89d/search?rangetype=relative&fields=message%2Csource&width=1920&highlightMessage=&relative=432000&q=%22Trying+to+proxy+URL%22

**Comment by Joozty** [comment]
@business-factory/pythonistas I guess we can merge it now. I cannot see any requests in the past 8 days.

**Pull request merged**